mix.exs

```diff
@@ -1,27 +1,14 @@
-defmodule Mix.Tasks.Compile.Exfswatch do
```

Comment:
I saw the commit, thank you ❤️ 
but I think we should change the order of compilers to `[:elixir, :app, :file_system]`, I just had a try and got `** (UndefinedFunctionError) function FileSystem.backend/0 is undefined (module FileSystem is not available)`, we should make sure we have `FileSystem` module compiled when run the `:file_system` compiler.

also, maybe move the `backend` function from `Utils` module to `FileSystem.Backend` module make more sence, could you please also include this change in your next pr?

Thank you!